frontend/src/locales/es.json

```diff
@@ -220,135 +220,133 @@
+    "We have received your statement": "Hemos recibido tu declaración",
+    "We are waiting for your trade counterpart statement. If you are hesitant about the state of the dispute or want to add more information, contact robosats@protonmail.com.": "Estamos esperando por tu contraparte, para que presente su declaración. Si estás dudoso sobre el estado de la disputa o quieres añadir más information, contacta a robosats@protonmail.com.",
+    "Please, save the information needed to identify your order and your payments: order ID; payment hashes of the bonds or escrow (check on your lightning wallet); exact amount of satoshis; and robot nickname. You will have to identify yourself as the user involved in this trade via email (or other contact methods).": "Por favor, guarda la información necesaria para identificar tu orden y tus pagos: ID de orden; claves del pago de la fianza o de los depósitos ( comprueba tu wallet lightning); cantidad exacta de satoshis; y nickname del robot. Tendrás que identificarte como el usuario involucrado en este intercambio via email (u otro método de contacto).",
+    "We have the statements": "Tenemos la declaración",
+    "Both statements have been received, wait for the staff to resolve the dispute. If you are hesitant about the state of the dispute or want to add more information, contact robosats@protonmail.com. If you did not provide a contact method, or are unsure whether you wrote it right, write us immediately.": "Ambas declaraciones se han recibido, espera a que el staff resuelva la disputa. Si estás dudoso sobre el estado de la disputa o quieres añadir información, contacta con robosats@protonmail.com. Si no diste un método de contacto, o estas inseguro de si lo escribiste bien, escribenos inmediatamente.",
+    "You have won the dispute": "Has ganado la diputa",
+    "You can claim the dispute resolution amount (escrow and fidelity bond) from your profile rewards. If there is anything the staff can help with, do not hesitate to contact to robosats@protonmail.com (or via your provided burner contact method).": "Puedes reclamar la cantidad de la resolución de la disputa (deposito y pago de fidelidad) desde las recompensas de tu perfil. Si hay algo que el staff pueda hacer, no dudes en contactar con robosats@protonmail.com (o a través del método de contacto de usar y tirar que especificaste).",
+    "You have lost the dispute": "Has perdido la diputa",
+    "Unfortunately you have lost the dispute. If you think this is a mistake you can ask to re-open the case via email to robosats@protonmail.com. However, chances of it being investigated again are low.": "Desafortunadamente has perdido la disputa. Si piensas que es un error también puedes pedir re-abrir el caso via email a robosats@protonmail.com. De todas formas, las probabilidades de ser investigado de nuevo son bajas.",
+    "INFO DIALOG - InfoDiagog.js": "Información de la App y terminos de uso",
```

Comment:
This line must be empty as the original.

Reply:
Please make the changes needed after forking from `spanish-translation`. If you need help let me know. I think it would take 1-2 hour max of my time to finish & polish the Spanish translation from where it is. 
Sentences at the top of the Trade box are too long! They can be shorter, no need to translate word by word.